Functions/InputOutput/xASL_io_PairwiseSubtraction.m

```diff
@@ -123,7 +123,7 @@ function xASL_io_PairwiseSubtraction(InputFile,outputPath,do_mask,switch_sign)
         matlabbatch{1}.spm.util.imcalc.outdir           = {path};
         matlabbatch{1}.spm.util.imcalc.options.dmtx     = 0;
         matlabbatch{1}.spm.util.imcalc.options.mask     = 0;
-        matlabbatch{1}.spm.util.imcalc.options.interp   = 1;
+        matlabbatch{1}.spm.util.imcalc.options.interp   = 4;
```

Comment:
Because we had spline in another interpolation function for registration. Cubic B-spline (option 4) is default for SPM resampling.  This one was linear because it was faster and this function was only used for intermediate images for registration/motion correction purposes only. Because I now built a new general subtraction function that will resample as well, it needs to always do it at high quality.

To be able to compare this with the previous code, the previous code (this function here) needs to have spline as well.

Reply:
Makes sense!